src/library.js

```diff
@@ -3503,7 +3503,7 @@ mergeInto(LibraryManager.library, {
 
   // Allocate memory for an mmap operation. This allocates space of the right
   // page-aligned size, and clears the allocated space.
-  $mmapAlloc__deps: ['$zeroMemory', '$alignMemory'],
+  $mmapAlloc__deps: ['$zeroMemory', '$alignMemory', 'emscripten_builtin_memalign'],
```

Comment:
I would guess this is the cause of the code size increases. Why is this necessary? I think perhaps we can leave this unchanged but make `FORCE_FILESYSTEM` in `library_wasmfs` include this method, as that is the only case it is needed? (since `FS.mmap` would only be added if `FORCE_FILESYSTEM`)